build.sbt

```diff
@@ -177,11 +191,13 @@ lazy val compiler = (project in file("compiler"))
 lazy val `sbt-exercise` = (project in file("sbt-exercise"))
   .settings(name := "sbt-exercise")
   .settings(
-    scalaVersion := "2.12.10",
+    scalaVersion := scala_212,
```

Comment:
Shouldn't be Scala 2.13?

Reply:
There's a dependency (`sbt-buildinfo`) which isn't updated to Scala 2.13 yet, so the plugin can't be updated to Scala 2.12.